src/main/java/org/spongepowered/api/item/inventory/custom/ContainerTypes.java

```diff
@@ -37,26 +37,31 @@
 
     // SORTFIELDS:ON
 
```

Comment:
`ContainerType`(s) shouldn't be in `custom` IMO. Since they are also used for vanilla.
This catalog should also be updated to 1.14: https://gist.github.com/Cybermaxke/c91032ba8571bddfaf3048250cbe6d10

Reply:
I agree with @Cybermaxke on this.